tests/pairlist/test_pairlist.py

```diff
@@ -302,6 +302,10 @@ def test_VolumePairList_refresh_empty(mocker, markets_empty, whitelist_conf):
     ([{"method": "VolumePairList", "number_assets": 5, "sort_key": "quoteVolume"},
       {"method": "ShuffleFilter"}],
      "USDT", 3),  # whitelist_result is integer -- check only length of randomized pairlist
+    # PerformanceFilter
+    ([{"method": "VolumePairList", "number_assets": 5, "sort_key": "quoteVolume"},
+      {"method": "PerformanceFilter"}],
+     "USDT", ['ETH/USDT', 'NANO/USDT', 'ADAHALF/USDT', 'ADADOUBLE/USDT']),
```

Comment:
It's still strange to me that changing this order fails tests.

Reply:
i'm not sure what you mean ... 
this test is there to verify the correct sequence of the output - so it should fail if it get's a wrong order back ...

Reply:
I had to place it in that specific order for the test to pass. If the order is different, it will get sorted into that order.

Perhaps that is expected for other reasons I'm missing.

Reply:
Oops... I see your [comment](https://github.com/freqtrade/freqtrade/pull/4009#discussion_r532053809) now and will investigate.